### PR TITLE
Adds integration tests with real browsers

### PR DESCRIPTION
**Pull request by pftg**

### CODE REVIEW

**Comment by pftg** [comment]
@donv please check, added tests in order to verify that js code is workable

**Comment by donv** [comment]
OK, looks promising.  I am unsure if the change in `lib/capybara/screenshot/diff/vcs.rb` is correct.

At least get the travis-ci checks green, then we can see.

**Comment by pftg** [comment]
@donv yep, will fix it today

**Comment by pftg** [comment]
@donv added fixes, should be verified soon https://travis-ci.org/github/donv/capybara-screenshot-diff/builds/747427376

File: lib/capybara/screenshot/diff/vcs.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@ module Vcs
 
         def restore_git_revision(name, target_file_name)
           redirect_target = "#{target_file_name} #{SILENCE_ERRORS}"
-          show_command = "git show HEAD~0:./#{Capybara::Screenshot.screenshot_area}/#{name}.png"
+          show_command = "git show HEAD~0:./#{Capybara::Screenshot.screenshot_area_abs}/#{name}.png"
```
Comment (donv) [inline lib/capybara/screenshot/diff/vcs.rb:12]: Are you sure this is right?

Comment (pftg) [reply lib/capybara/screenshot/diff/vcs.rb:12]: the problem that it tries to load `doc/screenshots` where `Rails.root` is not git root. So I just review code below, and it use abs path

Comment (pftg) [reply lib/capybara/screenshot/diff/vcs.rb:12]: or we need to cd to the `root` before to run this command

Comment (donv) [reply lib/capybara/screenshot/diff/vcs.rb:12]: But when you use the `_abs` path, you should not prefix with "./", right?

Comment (pftg) [reply lib/capybara/screenshot/diff/vcs.rb:12]: @donv I think I generated bad Rails.root, that's why I need to have `./` I'll recheck once more

Comment (pftg) [reply lib/capybara/screenshot/diff/vcs.rb:12]: and will try to add tests for this

Comment (pftg) [reply lib/capybara/screenshot/diff/vcs.rb]: @donv reverted my code, please recheck

**Comment by pftg** [comment]
@donv, please recheck

**Review by donv — APPROVED** [review]

**Pull request merged**